src/etc/inc/interfaces.inc

```diff
@@ -677,30 +677,24 @@ function interfaces_bridge_configure($checkmember = 0, $realif = "") {
 			if (empty($bridge['bridgeif'])) {
 				$bridge['bridgeif'] = "bridge{$i}";
 			}
-			if (!empty($realif) && $realif != $bridge['bridgeif']) {
+			if (!empty($realif) && ($realif != $bridge['bridgeif'])) {
 				continue;
 			}
-
-			if ($checkmember == 1) {
-				/* XXX: It should not be possible no? */
-				if (strstr($bridge['if'], '_vip')) {
-					continue;
-				}
```

Comment:
At the time this bit of code was added, that field did exist. Looks like it is `bridgeif` now. Though I don't think it's possible for a CARP VIP to get in there so maybe the check isn't necessary anymore. Back when that was added, CARP still worked like an interface and not like an IP alias as it does now.

Reply:
Interesting..
Will this PR be approved?